tests/CiIntegrationTest.php

```diff
@@ -219,22 +229,31 @@ public function getIntegrationCases()
                 array('0'),
                 '...',
             ),
+        );
+    }
+
+    public function testWithComposerLock()
+    {
+        $process = new Process('git check-ignore composer.lock', static::$fixtureDir);
+        if (0 === $process->run()) {
+            $this->markTestSkipped('The file "composer.lock" is ignored by git.');
```

Comment:
what about creating `.gitignore` file for our fixture repo that will unignore lock file ?
if not works, then I would change `git add .` to `git add . && git add composer.lock --force`

Reply:
:+1:  updated